package.json

```diff
@@ -7,10 +7,10 @@
   "scripts": {
     "prepare": "husky",
     "test:packages": "turbo run test --filter=./packages/*",
-    "build": "turbo run build",
     "build:packages": "turbo run build --filter=./packages/*",
     "lint:fix": "turbo run lint:fix",
-    "lint:fix-all": "prettier --write '{packages,apps, examples}/**/*.{js,jsx,ts,tsx,md,json}'",
+    "lint:packages": "turbo run lint --filter=./{packages,examples}/*",
+    "lint:fix-all": "prettier --write '{packages,apps,examples}/**/*.{js,jsx,ts,tsx,md,json}'",
```

Comment:
I suggest to rename the `lint:fix-all` to `format`, because the `prettier` is different from `eslint` and `typescript/eslint`